src/app/filecoin-plus/page.tsx

```diff
@@ -59,7 +59,18 @@ export default function FilPlus() {
         </div>
       </PageSection>
 
-      <PageSection kicker="Allocator Allocation" title="Become an Allocator">
+      <PageSection
+        kicker="Allocators"
+        title="Find an Allocator"
+        description="Discover the projects that apply for and receive DataCap, review the details of their applications, and find out who the allocators are."
+        image={graphicsData.imageFallback}
+        cta={{
+          href: FIL_PLUS_URLS.allocatorsList,
+          text: 'Allocators List',
+        }}
```

Comment:
Not sure if you intended to do this - This should link to the page create here https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/669, right?